TMDbLib/Client/TMDbClientMovies.cs

```diff
@@ -88,6 +89,9 @@ public async Task<Movie> GetMovieAsync(string imdbId, string language, MovieMeth
             if (language != null)
                 req.AddParameter("language", language);
 
+            if (includeImageLanguage != null)
```

Comment:
Would it make sense to default to DefaultLanguage here also?

Reply:
Yes and no. More results are definitely returned, but it doesn't work with `"en-US"`, only `"en"`.

Reply:
Ah - it would have to be different default value.. Could you add that, `DefaultImageLanguage` ?

Reply:
Sure https://github.com/LordMike/TMDbLib/pull/344/commits/224598343abdbefaf3075d096c372d1f5aa3f081